Update to Zewo 0.13

joernroeder opened the pull request:
Tries to update package to use Zewo 0.13, swift 3.0

Danappelxx committed:
Update PostgreSQL and SQL module layout

joernroeder committed:
update swift version, folder restructure

joernroeder committed:
Zewo/Core as only dependency

joernroeder committed:
`where` near generic will be deprecated -> change

joernroeder committed:
rename `Value.Error` due to ambiguity

joernroeder committed:
minor syntax / import updates

- changing `ErrorProtocol` to `Error`
- importing `Core`
- `StringLiteralConvertible` -> `ExpressibleByStringLiteral`
- string `split` -> `components`

robertjpayne commented on Sources/SQL/Core/ConnectionProtocol.swift:
```diff
@@ -36,13 +36,13 @@ public protocol ConnectionInfoProtocol {
     associatedtype InternalStatus
     associatedtype Result: ResultProtocol
-    associatedtype Error: ErrorProtocol, CustomStringConvertible
+    associatedtype ErrorAndStringConvertible: Error, CustomStringConvertible
```
Any reason you named this ErrorAndStringConvertible? It reads a lot nicer as `Error` even if it is CustomStringConvertible

Danappelxx replied:
I assume it's because `Error` is a reserved name by swift now. I'll change it to `ConnectionError`

Danappelxx replied:
Fixed :)

Danappelxx committed:
Remove duplicate import

Danappelxx committed:
Fix up a few things

- Fix .gitignore, .swift-version, .travis.yml, LICENSE
- Remove license headers from files
- Remove trailing whitespace from all files
- ErrorAndStringConvertible -> ConnectionError

Danappelxx commented:
I rebased on master and fixed a few things. Your part looks good! Once CI passes I'll merge 👍

Danappelxx committed:
Merge branch 'master' into 0.13

Danappelxx commented:
Excellent! Merging
